styles.css

```diff
@@ -20,61 +19,81 @@ h1,
   color: #e9e9f3;
 }
 
+.show-starred-ideas {
+  padding: 0px;
+  margin: auto;
+  border: solid #353567 3px;
+  box-shadow: none;
+  border-radius: 0;
+  border: 0;
+}
+
 aside {
-  border-style: solid;
+  font-family: open sans, bold;
```

Comment:
I discovered for the font we actually need "font-family: "Open Sans" sans-serif;" 
And I believe bold is a separate property called font-weight.